pkg/fanal/analyzer/sbom/testdata/kbom.json

```diff
@@ -0,0 +1,434 @@
+{
+  "$schema": "http://cyclonedx.org/schema/bom-1.5.schema.json",
+  "bomFormat": "CycloneDX",
+  "specVersion": "1.5",
+  "serialNumber": "urn:uuid:e2daaea6-d96f-4b84-960c-0d72c348cd23",
+  "version": 1,
+  "metadata": {
+    "timestamp": "2023-09-29T06:25:00+00:00",
+    "tools": [
+      {
+        "vendor": "aquasecurity",
+        "name": "trivy",
+        "version": "0.45.1-15-g7bbd0d097"
+      }
+    ],
+    "component": {
+      "bom-ref": "pkg:k8s/k8s.io/kubernetes@1.27.4",
+      "type": "platform",
+      "name": "k8s.io/kubernetes",
+      "version": "1.27.4",
+      "purl": "pkg:k8s/k8s.io/kubernetes@1.27.4",
+      "properties": [
+        {
+          "name": "aquasecurity:trivy:resource:Name",
+          "value": "minikube"
+        },
+        {
+          "name": "aquasecurity:trivy:resource:Type",
+          "value": "cluster"
+        }
+      ]
+    }
+  },
+  "components": [
+    {
+      "bom-ref": "5262e708-f1a3-4fca-a1c3-0a8384f7f4a5",
+      "type": "operating-system",
+      "name": "ubuntu",
```

Comment:
I created this KBOM file using Trivy.
Why don't we include OS packages like in table format?
```
➜  trivy -d k8s cluster --report all
...
registry.k8s.io/kube-proxy:v1.27.4 (debian 11.6)
================================================
Total: 36 (UNKNOWN: 0, LOW: 22, MEDIUM: 12, HIGH: 2, CRITICAL: 0)
```